app-desktop/build.gradle.kts

```diff
@@ -23,9 +23,10 @@ kotlin {
         named("jvmMain") {
             dependencies {
                 implementation(compose.desktop.currentOs)
-                implementation(projects.ui.timeline)
+                implementation(projects.ui.splash)
```

Comment:
Why does the app only depend on the splash screen? Looks like the ui-modules are not decoupled anymore.

Reply:
Hmm yeah not sure how to go about this quite yet.. if I move the root graph code into the apps, the logic gets duplicated, but then I would be able to keep UI modules all decoupled and independent. 

If I want to commonize the root graph, it requires the root module (in this case splash), depending on starting points for other navigation trees (sign-up/login flow when user is logged out and timeline when user is logged in)